src/utils/appData.ts

```diff
@@ -21,12 +26,57 @@ async function getValidator(): Promise<{ ajv: Ajv; validate: ValidateFunction }>
   return { ajv, validate }
 }
 
-export async function validateAppDataDocument(appDataDocument: any): Promise<ValidationResult> {
+function buildCidInstance(hash: string) {
+  const cidVersion = 0x1 //.toString(16) //cidv1
+  const codec = 0x70 //.toString(16) //dag-pb
+  const type = 0x12 //.toString(16) //sha2-256
+  const length = 32 //.toString(16) //256 bits
```

Comment:
Those are debug/failed attempt artifacts

```suggestion
  const cidVersion = 0x1 //cidv1
  const codec = 0x70 //dag-pb
  const type = 0x12 //sha2-256
  const length = 32 //256 bits
```